src/Parser/Paladin/Protection/Modules/Talents/Seraphim.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { formatPercentage } from 'common/format';
+
+/**
+ * Consumes up to 2 SotR charges to provice 5500 Haste+Vers+Mastery+Crit for 8sec per consumed charge
+*/
+
+//ToDo: remove possible SotR casts from Ability-Tab that were used by Seraphim
```

Comment:
You can just do `this.spellUsable.beginCooldown(SPELLS.SHIELD_OF_THE_RIGHTEOUS.id, event.timestamp);` for each charge spent by Seraphim